test/e2e/metering_manual_install_tests.go

```diff
@@ -212,3 +215,76 @@ func customNodeSelectorFunc(ctx *deployframework.DeployerCtx) error {
 
 	return nil
 }
+
+// createMySQLDatabase is a test helper function that is
+// responsible for initializing an ephemeral mysql database
+// that will be used as the underlying Hive metastore database
+// for an individual Metering test installation.
+func createMySQLDatabase(ctx *deployframework.DeployerCtx) error {
+	const (
+		mysqlNamespace     = "mysql"
+		mysqlLabelSelector = "db=mysql"
+	)
+	_, err := ctx.Client.CoreV1().Namespaces().Get(context.Background(), mysqlNamespace, metav1.GetOptions{})
+	if err != nil && !apierrors.IsNotFound(err) {
+		return nil
+	}
+	if apierrors.IsNotFound(err) {
+		n := &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: mysqlNamespace,
+				Labels: map[string]string{
+					"name": ctx.Namespace + "-" + testingNamespaceLabel,
+				},
+			},
+		}
+		_, err = ctx.Client.CoreV1().Namespaces().Create(context.Background(), n, metav1.CreateOptions{})
+		if err != nil {
+			return err
+		}
+		ctx.Logger.Debugf("Created the %s namespace", mysqlNamespace)
+	}
+
+	cmd := exec.Command(
+		"oc",
+		"-n", mysqlNamespace,
+		"new-app",
+		"--image-stream", "mysql:5.7",
```

Comment:
Would this work a more recent MySQL version but it's not available in CI?

Reply:
Nope, we have [open BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1838802) for Metering where we can't work with mysql8. This kind of e2e test would be nice in verifying that the proposed fix (updating the hadoop image to pull an updated mysql connector jar) would still work with mysql5.7, in addition to a more recent release.